shallow_backup/reinstall.py

```diff
@@ -73,6 +73,14 @@ def reinstall_configs_sb(configs_path):
 		elif os.path.isfile(path_to_backup):
 			copyfile(path_to_backup, dest_path)
 
+	# reinstall crontab
+	with open(os.path.join(configs_path, "crontab.txt"), "r") as f:
+		for x in f:
+			# the replace sanitizes the crontab line of any present " characters.
+			sanitized = x.replace('"', '\\"')
+			cmd = f"(crontab -l ; echo \"{sanitized}\") | sort - | uniq - | crontab -"
```

Comment:
I'm also not convinced this is the best way to do this. This escaping may not work properly in all cases. Make sure to test it with some complex commands that require proper escaping.